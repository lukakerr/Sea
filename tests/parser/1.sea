fn main {} (
  ret (1 + (3 + 5))
)