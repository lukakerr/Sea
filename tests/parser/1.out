Main (Return (App (App (Prim Plus) (Const (Number 1))) (App (App (Prim Plus) (Const (Number 3))) (Const (Number 5)))))
